src/components/StyledList/StyledList.tsx

```diff
@@ -35,44 +37,45 @@ export const StyledList: ChakraComponent<
   >,
   React.PropsWithChildren<StyledListProps>
 > = chakra(
-  forwardRef<HTMLDivElement & HTMLUListElement, StyledListProps>(
-    (props, ref?) => {
-      const {
-        children,
-        className,
-        id,
-        listItems = [],
-        style = "capped",
-        textSize = "default",
-      } = props;
-      const styles = useStyleConfig("StyledList", {
-        textSize,
-        variant: style,
-      });
-      let finalChildren;
+  forwardRef<
+    HTMLDivElement & HTMLUListElement,
+    React.PropsWithChildren<StyledListProps>
+  >((props, ref?) => {
+    const {
+      children,
+      className,
```

Comment:
Why is this still included? Where is `...rest`?

Reply:
Removed `className`, added `...rest`.